src/storm/utility/ConstantsComparator.h

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#include "storm/adapters/IntervalAdapter.h"
```

Comment:
Is it possible to keep the include only in the cpp file? This avoids costly includes of carl in header files.